modules/cli/pages/index.adoc

```diff
@@ -131,6 +131,23 @@ There are a number of `skupper` options regarding authentication for the console
 Defaults to `admin`.
 `--console-password <password>`:: Password for the console user when authentication mode is set to `internal`.
 If not specified, a random passwords is generated.
+--
+
+
+
+* Configuring service access
++
+--
+----
+$ skupper init --create-network-policy
+----
+
+By default, services exposed on a site in a cluster may become accessible on all namespaces on a remote cluster which has a site initialized. 
```

Comment:
Services in a given namespace may be accessible to pods in other namespaces on that cluster by default, exposing any services accessible through skupper to pods in namespaces not directly connected to the skupper network. This setting applies a network policy to restrict access to services accessed via skupper to those pods in the namespace in which skupper is configured.

Reply:
tried to incorporate your suggestion into doc, while sticking to 'service network', lemme know if it works @grs